diff --git a/Data/User/Config/GFX.ini b/Data/User/Config/GFX.ini
index f2699c36b..a6ca8f1b3 100644
--- a/Data/User/Config/GFX.ini
+++ b/Data/User/Config/GFX.ini
@@ -24,11 +24,11 @@ WaitForCachedHiresTextures = False
 [Hacks]
 EFBAccessEnable = False
 EFBScaledCopy = False
-FullAsyncShaderCompilation = True
+FullAsyncShaderCompilation = False
 [Hardware]
 VSync = False
 [Enhancements]
-Tessellation = True
+Tessellation = False
 ScalingShader = 
 FilteringMode = 1
 MaxAnisotropy = 0
